packages/app-elements/src/providers/TokenProvider/TokenProvider.tsx

```diff
@@ -104,6 +104,14 @@ export interface TokenProviderProps {
    * This methods will also be returned within the context as part of `TokenProviderValue` object.
    */
   onAppClose?: () => void
+  /**
+   * Optional. Extra information to pass from the dashboard or when the app is initialized.
+   * They will be received on app init and make available in the TokenProvider > settings context.
+   */
+  metadata?: {
```

Comment:
I don't know if `metadata` could be the right name, what do you think about `extra`, or `settings`?

Reply:
changed!